README.md

```diff
@@ -22,17 +22,45 @@ This is a Ruby on Rails application.
 
 ### Running the application
 
+The first time you run this application you'll want to set you
+[permissions](#permissions).
+
+Then, if you are running on the [GOV.UK development VM][dev-vm]:
+
 ```
+cd /var/govuk/govuk-puppet/development-vm
 bowl content-publisher
 ```
 
+Otherwise, on your machine:
+
+```
+./startup.sh
+```
+
 ### Running the test suite
 
 ```
 yarn install
 bundle exec rake
 ```
 
+### Permissions
+
+Functionality of this application is enabled with permissions. There is a
```

Comment:
Could we just tell them to run `rake db:seed` and put the permissions in there?

Reply:
The problem we've got is that if you clone data then you won't have run (or plan to run) db:seed.

Reply:
Yup. I think I forgot the seed deliberately doesn't have the 'debug' permission, so we need a separate task for it.